docs/manual/objects.rst

```diff
@@ -72,18 +72,35 @@ a point object.
 Insert Ellipse
 ~~~~~~~~~~~~~~
 
-Shortcut: ``C``
+Shortcut: ``C`` |insert-ellipse|
 
 Ellipses work the same way as `rectangles <#insert-rectangle>`__, except
 that they are rendered as an ellipse. Useful for when your area or
 collision shape needs to represent a circle or ellipse.
 
+
+.. raw:: html
+
+   <div class="new">Since Tiled 1.12</div>
+
+
+.. _insert-capsule-tool:
+
+Insert Capsule
+~~~~~~~~~~~~~~
+
+No default shortcut |insert-capsule|
+
+Capsules work the same way as `rectangles <#insert-rectangle>`__, except
+that they are rendered as an capsule. Useful for when your area or
```

Comment:
Tiniest nitpick but this should be "a capsule"

Reply:
Thanks! That's what I get from simply copy/pasting and replacing "ellipse". :P